content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -134,11 +134,12 @@ path types:
   to the list of labels in the path split by the `/` separator. A request is a
   match for path _p_ if every _p_ is an element-wise prefix of _p_ of the
   request path.
-    {{< note >}}
-    If the last element of the path is a substring of the
-    last element in request path, it is not a match (for example:
-    `/foo/bar` matches`/foo/bar/baz`, but does not match `/foo/barbaz`).
-    {{< /note >}}
+
+{{< note >}}
+If the last element of the path is a substring of the
+last element in request path, it is not a match (for example:
+`/foo/bar` matches`/foo/bar/baz`, but does not match `/foo/barbaz`).
+{{< /note >}}
```

Comment:
This looks like the wrong fix to me. I think the note should remain part of the bullet for _`Prefix`_.

Reply:
Seems like when I build this whole repo using hugo version `0.57.2` it looks okay without any changes, however I notice there's many broken page after we're upgrading to `0.70.0`
This is including the Deployment page.